test/test_grocy.py

```diff
@@ -1,37 +1,78 @@
 from unittest import TestCase
 from unittest.mock import patch, mock_open
 
+import json
 from datetime import datetime
 from requests.exceptions import HTTPError
 import responses
 from pygrocy import Grocy
-from pygrocy.grocy import Product
-from pygrocy.grocy import Group
-from pygrocy.grocy import ShoppingListProduct
-from pygrocy.grocy_api_client import CurrentStockResponse, GrocyApiClient
+from pygrocy.grocy import Chore, Product, Group, ShoppingListProduct
+
+from pygrocy.grocy_api_client import CurrentStockResponse, GrocyApiClient, TransactionType, UserDto, ChoreDetailsResponse, ProductData
 from test.test_const import CONST_BASE_URL, CONST_PORT, CONST_SSL
 
 class TestGrocy(TestCase):
 
     def setUp(self):
-        self.grocy = Grocy(CONST_BASE_URL, "api_key")
-        self.grocy = None
+        self.grocy_regular = Grocy(CONST_BASE_URL, "api_key")
         self.grocy = Grocy(CONST_BASE_URL, "demo_mode",  verify_ssl = CONST_SSL, port = CONST_PORT)
+        self.base_url = f"{CONST_BASE_URL}:{CONST_PORT}/api"
+        self.date_test = datetime.strptime("2019-05-04 11:31:04", '%Y-%m-%d %H:%M:%S')
 
     def test_init(self):
-        assert isinstance(self.grocy, Grocy)
+        self.assertIsInstance(self.grocy, Grocy)
         
     def test_get_chores_valid(self):
         chores = self.grocy.chores(get_details=True)
         
-        assert isinstance(chores, list)
-        self.assertEqual(len(chores), 6)
-        assert chores[0].id == 1
-        assert chores[1].id == 2
-        assert chores[2].id == 3
-        assert chores[3].id == 4
-        assert chores[4].id == 5
-        assert chores[5].id == 6
+        self.assertIsInstance(chores, list)
+        self.assertGreaterEqual(len(chores), 1)
```

Comment:
is it possible to change these into absolute values, switching from `GreaterEqual` to `Equal`...

Reply:
Sure but it's not a mocking response.
Should I restore to a mocking test ?
For tests through real server I suggest "relative" (not sure about this word) tests.
Let me know.

Reply:
I thought we had control about the test data. If that's the case we should pin down the assertions as much as possible.

Reply:
It's demo data, I don't know what exactly is in. But it covers most of case.
But it was the purpose of using a real Grocy instance ?

Reply:
Oh it's fine then.